### PR TITLE
Fix test regression due to multi service manifest change

### PR DESCRIPTION
**Pull request by azymnis**
Addresses issue #272

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

**Comment by azymnis** [comment]
@ittaiz hmm the tests fail, let me take a look at what is going on here

**Comment by johnynek** [comment]
test this please

(cibot message)

**Comment by johnynek** [comment]
@azymnis can you merge master in, it looks like this is off an old branch

File: test_run.sh [hunk 1]
```diff
@@ -25,12 +25,16 @@ fi
 echo "$_md5_util"
 }
 
+non_deploy_jar_md5_sum() {
+  find bazel-bin/test -name "*.jar" ! -name "*_deploy.jar" | $(md5_util)
```
[the remark this answers is not in the document]
Comment (johnynek) [reply test_run.sh]: okay, sorry for being out of the loop. I get it now: we are explicitly requesting `_deploy.jar` in one branch but not the other. I guess this is okay, but I'd rather include it and make sure the deploy jars show up in both runs.

[the remark this answers is not in the document]
Comment (ittaiz) [reply test_run.sh]: That was my thought as well but, like I commented in the issue, adding this deploy jar seems a bit like a red herring to me. That's why I thought we should exclude it and later add building one of the small existing binaries as a deploy jar.
I don't have a strong preference so I'm ok with building this one here as well.

**Comment by johnynek** [comment]
I'm happy to merge and we can follow up to improve deploy reproducibility testing.

**Comment by ittaiz** [comment]
Excellent.
Thanks Argyris for following through.

**Comment by azymnis** [comment]
Thanks @johnynek @ittaiz 

I think there's some issue with my master branch. I'll deal with this on my own fork

**Comment by ittaiz** [comment]
An issue related to this PR?

**Comment by azymnis** [comment]
@ittaiz no, just a bad merge on my side. Fixed now

**Pull request merged**